wasi/wasi.go

```diff
@@ -50,7 +50,7 @@ type API interface {
 	//                     offset that begins "bc" --+
 	//
 	// Note: FunctionArgsGet documentation has an example of this signature in the WebAssembly 1.0 (MVP) Text Format.
-	// See ArgsSizesGet
```

Comment:
oh this was intended as it is telling you to look at the other function which is relevant to this one. If we make the change here, it would be a loop!

Reply:
Ah, ok, I see.
 Will cancel PR